var/spack/repos/builtin/packages/dray/package.py

```diff
@@ -0,0 +1,330 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import os
+import socket
+
+import llnl.util.tty as tty
+from os import environ as env
+
+
+def cmake_cache_entry(name, value, vtype=None):
+    """
+    Helper that creates CMake cache entry strings used in
+    'host-config' files.
+    """
+    if vtype is None:
+        if value == "ON" or value == "OFF":
+            vtype = "BOOL"
+        else:
+            vtype = "PATH"
+    return 'set({0} "{1}" CACHE {2} "")\n\n'.format(name, value, vtype)
+
+
+class Dray(Package, CudaPackage):
+    """High-Order Mesh Ray Tracer."""
+
+    homepage = "https://github.com/LLNL/devil_ray"
+    git      = "https://github.com/LLNL/devil_ray.git"
+    url      = "https://github.com/LLNL/devil_ray/releases/download/v0.1.2/dray-v0.1.2.tar.gz"
+
```

Comment:
@mclarsen @cyrush do you want to list yourselves as maintainers here?
```suggestion
    maintainers = ['cyrush', 'mclarsen']

```

Reply:
done